assets/js/modules/tagmanager/hooks/useExistingTagEffect.js

```diff
@@ -36,10 +36,16 @@ export default function useExistingTagEffect() {
 		select( MODULES_TAGMANAGER ).getContainerID()
 	);
 
+	const initialContainerID = useRef( containerID );
```

Comment:
It might be a little clearer to rename this and make it a boolean:

```js
	const skipEffect = useRef( !! containerID );
	
	// ...
	
	useEffect( () => {
		if ( skipEffect.current ) {
			skipEffect.current = false;
			return;
		}
```